ideascube/models.py

```diff
@@ -244,6 +244,14 @@ def index_strings(self):
         ('female', _('Female')),
     )
 
+    DISABILITY_CHOICES = (
+        ('visual', _('Visual')),
+        ('auditive', _('Auditive')),
+        ('physical', _('Physical')),
+        ('cognitive', _('Cognitive')),
+        ('mental', _('Mental')),
```

Comment:
I don't really know either. This list comes from Sergio, BSF project manager in Colombia.
They are supposed to be generic categories, not specific disabilities.
I suppose, the categories may vary according to the specialist answering, and I'm not sure we have a better person than Sergio to arbitrate.
The immediate need is to match with Llave del saber (which does not know about "mental", by the way; this has been added by Sergio). I think we are good to go, given we can still add more in the future.